docs/rgw_setup.md

```diff
@@ -0,0 +1,136 @@
+# Setting up Ceph Object Gateway for different RPC Use Cases
+
+Ceph Object Gateway(rgw) is an object storage interface built on top of librados to provide applications with a RESTful gateway to Ceph Storage Clusters. Ceph Object Storage supports two interfaces  S3 and OpenStack Swift.   
+
+Complete documentation for the Ceph Object Gateway can be found (here.)[http://docs.ceph.com/docs/master/radosgw/]
+
+## Role of rpc-ceph
+The purpose of (rpc-ceph)[https://github.com/rcbops/rpc-ceph] is to provide a consistant versioned set of processes, variables and versioned software for RPC's use.  
+
+### List of Use Cases
+
+* General Installation using ceph-ansible
+* Configuring rgw to use a Stand alone instance of OpenStack Keystone
+* Configuring rgw to use KeyStone and Interact with an OpenStack environment in place of OpenStack Swift
+
+
+
+## Installation of rgw only
+To Install rgw **ONLY** and **NOT** use any of rpc-ceph configured varibles 
+
+### To install rgw addtions to the inventory file.   
+
+Add the **rgw** group with a list of hosts into the ceph environment's inventory file. 
+For example:
+```bash
+[rgw]
+ceph-rgw01
+ceph-rgw02
+ceph-rgw03
+```
+
+### Installation variables
```

Comment:
There quite a few assumptions made to the configuration in rgws.yml file.   The port is set to 8080, etc.     IMO the first use case is more for someone experimenting or testing but we can certainly remove it if you think it isn't necessary

Reply:
The only default vars we set for radosgw, which are needed:
radosgw_address_block: "{{ public_network }}"
radosgw_civetweb_bind_ip: "{{ hostvars[inventory_hostname]['ansible_all_ipv4_addresses'] | ipaddr(radosgw_address_block) | first }}"
radosgw_civetweb_port: 8080

The port is the default value for ceph anyway, so if you don't specify it you should still get 8080 (or it will fail because its not specified), the other 2 are required to tell radosgw where to listen, regardless of keystone integration or not. 
All the other vars in rgws.yml won't be used unless you set radosgw_keystone: True, or you are using some old version of ceph-ansible which isn't shipped in rpc-ceph.

Reply:
I completely removed the first use case to